.bazelrc

```diff
@@ -52,9 +52,6 @@ build:ios_multi_arch_test --ios_minimum_os=10.2
 # to exemplify fat binaries on the latest version
 build:ios_multi_arch_test --ios_multi_cpus=x86_64
 
-# - Remote cache configuration
-build:ci_with_caches --config=ci --config=remote_cache
```

Comment:
This config was previously used by all jobs, seemed redundant to define another `ci` config so just using `ci` with `--remote_cache` by default now